packages/ui2/src/components/button.tsx

```diff
@@ -1,30 +1,54 @@
 import { type VariantProps, cva } from "class-variance-authority";
 
+import { cn } from "../lib/utils.js";
```

Comment:
This is causing ts compilation error and should be imported like this:
```
import { cn } from "@recallnet/ui2/lib/utils"
```
This is a little surprising to me that what you have doesn't work, but I think it has to do with how we are transpiling the ui2 code from the web app and how we have the tsconfig path remapping setup in ui2. I copied this setup from shadui, so it must not be too bad :)

Reply:
yeah, the build wasn't working but the dev env was. My editor setup auto imports files without taking into account the tsconfig mapping